app/views/coronavirus_pages/_actions.html.erb

```diff
@@ -1,17 +1,15 @@
 <div class="app-side">
   <div class="app-side__actions">
-    <%= render "govuk_publishing_components/components/button", {
-      text: "Publish",
-      href: "#"
-    } %>
+
+    <%= link_to "Publish", coronavirus_page_publish_path(@coronavirus_page.slug), method: :post, class: "govuk-button" %>
 
     <%= render "govuk_publishing_components/components/button", {
```

Comment:
It there a reason why we switched the publish button to be a `link_to`, but left this as a button component?

Reply:
I couldn't figure out how to add a POST method to the component - I'm not sure you can give it a method?? I'll check with dilwoar and reuse the component if it's poss.